go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/Financial-Times/mongo-hot-backup
```

Comment:
If the version is going to be `v3.0.0` then the module path should be ending with `/v3`.

Reply:
It is removed, thanks for the explanation.